### DISCUSSION TITLE
Is there a workaround to import Observing Lists in V 23.2?

### DISCUSSION THREAD

**Discussion by Menno5** [Stellarium/stellarium#3312] [Q&A] [answered]
I presume this is the same as #3206 
I exported my Observer Lists in V 23.1 and am now trying to import it into V 23.2 but nothing happens.
I checked the log and there there is only one line: **[ObservingList] conversion error**
Weird thing is that when I check the _observingList.json_ file in _AppData\Roaming\Stellarium\data_ , it does contain all the data that I did import.

Is there a workaround for now to get it showing in V 23.2?

**Comment by gzotti** [comment, marked as answer]
I had wanted to complete the ObsLists issues by 23.2 in #3207. @alex-w wanted to extend it even more with import/export of single lists, other file endings etc., however real life interfered. It has been rescheduled to 23.3. If you can compile yourself, you can give this branch a try.

**Comment by Menno5** [comment]
Complete compiling noob here, so no ;)
I keep an eye out on the weekly snapshots instead then.
